engine/menus/save.asm

```diff
@@ -161,6 +161,8 @@ AddHallOfFameEntry:
 	ld bc, wHallOfFamePokemonListEnd - wHallOfFamePokemonList + 1
 	call CopyBytes
 	call CloseSRAM
+; Causes the emulator to set sMobileEventIndex and sMobileEventIndexBackup to $b
```

Comment:
Mention `MOBILE_EVENT_OBJECT_GS_BALL` instead of `$b`. I'm not sure this constant used to exist, but it does now.

Reply:
https://github.com/pret/pokecrystal/pull/882/commits/4059f57948a750d1b1262152136a9be1e53b4969 should resolve this.